packages/frint-react/src/components/Region.spec.js

```diff
@@ -353,4 +353,53 @@ describe('frint-react › components › Region', function () {
     // rootApp should not have the instance any more
     expect(window.app.getAppInstance('App', 'todo-item', 'todo-item-1')).to.equal(null);
   });
+
+  it('should accept className and pass down to rendered component', () => {
```

Comment:
in tests, we usually avoid arrow functions unless they are single-lined, or needs to access parent scope.

can be just `function () {` here.